dynamixel_mx:
  port: "/dev/ttyACM0"
  baudrate_board: 115200
<<<<<<< HEAD
  init_pose: 1327 #45째
  final_pose: 2130 #-25째
=======
  init_pose: 1327    #-25째
  final_pose: 2130 #45째  
>>>>>>> 9b5e88b9d75cc04ddf338382aa46e38ecda029dd
  step_org: 10
  step_scan: 3
  delta_error: 3
